samples/dymaptic.GeoBlazor.Core.Sample.TokenRefresh/dymaptic.GeoBlazor.Core.Sample.TokenRefresh.Client/Pages/Home.razor

```diff
@@ -0,0 +1,132 @@
+    <div class="row">
+        <div class="col-12">
+            <h1 class="mb-4">GeoBlazor Token Refresh Sample</h1>
+            
+            <!-- Documentation Section -->
+            <div class="documentation-section mb-4">
+                <p><strong>Token Authentication Process</strong></p>
+                <p>
+                    <strong>This sample demonstrates how to implement automatic token refresh in GeoBlazor applications 
+                    using server-side authentication with ArcGIS Portal or ArcGIS Online.</strong>  
+                    For more details on setting up OAuth credentials in Portal or ArcGIS Online please refer to the following tutorial: 
+                    <a href="https://developers.arcgis.com/documentation/security-and-authentication/app-authentication/tutorials/create-oauth-credentials-app-auth" target="_blank">Create OAuth credentials for app authentication</a>
+                </p>
+
+                <p><strong>1. Configuration Setup</strong></p>
+                <p>
+                    Configure your ArcGIS authentication settings in <code>appsettings.json</code> or user secrets:
+                </p>
+                
+                <div class="alert alert-info">
+                    <strong>Current Server Configuration:</strong>
+                    <pre><code>@_currentConfig</code></pre>
+                </div>
+
+                <div class="alert alert-warning">
+                    <strong>Portal vs ArcGIS Online Configuration</strong>
+                    <ul>
+                        <li><strong>ArcGIS Online:</strong> Leave <code>ArcGISPortalUrl</code> set to <code>https://www.arcgis.com/</code> or leave empty since this is the default value in code</li>
+                        <li><strong>Enterprise Portal:</strong> Set <code>ArcGISPortalUrl</code> to your portal URL and <strong>include <code>/portal/</code></strong> at the end as shown above</li>
+                    </ul>
+                </div>
+
+                <p><strong>2. Authentication Flow</strong></p>
+                <p>
+                    The authentication process involves two main components working together:
+                </p>
+                
+                <p><strong>ArcGisAuthService.cs</strong></p>
+                <ul>
+                    <li>Makes HTTP requests to ArcGIS REST API</li>
+                    <li>Handles client credentials flow</li>
+                    <li>Caches tokens to local file</li>
+                    <li>Manages token expiration</li>
+                </ul>
+
+                <p><strong>AuthenticationManager.cs/.ts</strong></p>
+                <ul>
+                    <li>C# wrapper around TypeScript implementation</li>
+                    <li>Integrates with ArcGIS Maps SDK for JavaScript</li>
+                    <li>Registers tokens with ArcGIS Identity Manager</li>
+                    <li>Provides authentication state management</li>
+                </ul>
```

Comment:
This is confusing to a GeoBlazor NuGet package _user_, because they won't be writing `AuthenticationManager`, just using it. We should distinguish between what is in this sample and is their responsibility (`ArcGISAuthService`) and what is built into GeoBlazor (`AuthenticationManager`).

Reply:
And no need to talk about the .cs/.ts files within GeoBlazor.